WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -0,0 +1,77 @@
+package org.wordpress.android.ui.people.utils;
+
+import com.android.volley.VolleyError;
+import com.wordpress.rest.RestRequest;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.AppLog.T;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PeopleUtils {
+
+    public static void fetchUsers(String siteID, final int localTableBlogId, final PeopleUtils.Callback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null) {
+                    try {
+                        JSONArray jsonArray = jsonObject.getJSONArray("users");
+                        List<Person> people = peopleListFromJSON(jsonArray, localTableBlogId);
+                        PeopleTable.savePeople(people);
```

Comment:
I am not completely sold on this because I don't like the idea of an Activity handling the DB changes. How about if I change it to following methods (parameters might change a bit):

```
public static void fetchUsersAndSaveToDB(String siteID, final int localTableBlogId, final PeopleUtils.Callback callback) {}
private static void fetchUsers(String siteID, final int localTableBlogId, final PeopleUtils.Callback callback) {}
```
I could also add a method `private static void saveUsersToDB(List<Person> peopleList){}` but then I'll need to make sure it's run in the background and it might be an overkill since we can just do it in the method's callback before it makes it to the UI callback.

My idea is basically keeping DB and Network functions in the Utils class and the fetch function should still be unit testable and the DB one is already a separate method we are calling.

What do you think?